src/main/java/com/rarchives/ripme/ripper/rippers/CfakeRipper.java

```diff
@@ -50,6 +50,7 @@ public Document getFirstPage() throws IOException {
         @Override
         public Document getNextPage(Document doc) throws IOException {
             // Find next page
+            @SuppressWarnings("unused")
```

Comment:
Don't suppress warnings if possible. Figure out what was intended and update the code accordingly. In this case it might simply be that the variable is unused and was left behind by a past change, and should now be removed. That looks to be the case here.